addons/l10n_tr/demo/demo_company.xml

```diff
@@ -28,7 +28,7 @@
     </function>
 
     <function model="account.chart.template" name="try_loading">
-        <value eval="[ref('l10n_tr.chart_template_common')]"/>
+        <value eval="[ref('l10n_tr.chart_template_7b')]"/>
```

Comment:
Why do you change the name of the COA? Even if it's not perfect, I prefer to avoid such change if not necessary

Reply:
Because there are two official CoA in Turkey, before we were just ignoring one of them, and the naming would be weird otherwise. In other words,  `chart_template_common` does not have the complete accounts, it's children 7b and 7a or complete.

Reply:
ok then :)